src/main.rs

```diff
@@ -249,9 +253,11 @@ fn create_replicator(
     );
     Arc::new(replicator)
 }
-
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn Error>> {
+    crypto::CryptoProvider::install_default(ring::default_provider())
+        .expect("Failed to install rustls crypto provider");
```

Comment:
Can this fail? Should we only do this if we need to replicate?

Reply:
fixed